packages/marko-web-identity-x/browser/index.js

```diff
@@ -0,0 +1,10 @@
+const IdentityXAuthenticate = () => import(/* webpackChunkName: "identity-x-authenticate" */ '@base-cms/identity-x/browser/authenticate.vue');
+const IdentityXLogout = () => import(/* webpackChunkName: "identity-x-logout" */ '@base-cms/identity-x/browser/logout.vue');
+const IdentityXForm = () => import(/* webpackChunkName: "identity-x-form" */ '@base-cms/identity-x/browser/form.vue');
+
+export default (Browser) => {
+  const { EventBus } = Browser;
+  Browser.register('IdentityXAuthenticate', IdentityXAuthenticate, { on: { action: EventBus } });
```

Comment:
How are the actions named? They'll likely need to be prefixed with `identity-x-`